backend/Utils/EmailUtils.ts

```diff
@@ -2,9 +2,32 @@ import google from "googleapis";
 import path from "path";
 import { encode } from "js-base64";
 import dotenv from "dotenv";
+import jwt from "jsonwebtoken";
 
-async function makeEmail({ to, from, subject }) {
-  const message = `This is a test email for reset password.`;
+dotenv.config();
```

Comment:
Does `dotenv.config()` need to be called here?

Reply:
No we should remove this